data/css/modules/card/_knowledgeDocument.scss

```diff
@@ -154,10 +154,19 @@ $toc-align-padding: 15px;   /* This aligns the document title to the TOC
         &:hover {
             color: #54595a;
         }
-    }
+    }    
+}
 
-    .composite &__entry {
-        font-size: 21.1px;  /* 18 px spec * 88% because spec was at 110% scale not 125% */
-        background-color: transparentize(black, 1 - 0.8);
+.composite {
+    .InArticleSearch {
+        &__entry {
+            font-size: 21.1px;  /* 18 px spec * 88% because spec was at 110% scale not 125% */
+            background-color: transparentize(black, 1 - 0.8);
+        }
     }
+
+    /*
+    TableOfContents NOTE: 
+    Some TableOfContents composite styling is in the .TableOfContents class, as &--collapsed. 
```

Comment:
It would be more helpful to comment this in the corresponding place where the styling is. I wonder why we have these separated.